src/pages/staking/Staking.vue

```diff
@@ -36,21 +36,21 @@
                         :to="{ hash: '#stake'}"
                         exact
                         push
-                        label="Stake"
+                        :label="$t('pages.staking.stake')"
                     />
                     <q-route-tab
                         name="unstake"
                         :to="{ hash: '#unstake'}"
                         exact
                         push
-                        label="Unstake"
+                        :label="$t('pages.staking.unstake')"
                     />
                     <q-route-tab
                         name="withdraw"
                         :to="{ hash: '#withdraw'}"
                         exact
                         push
-                        label="Withdraw"
+                        :label="$t('pages.staking.withdraw')"
                         :alert="showWithdrawNotification ? 'green' : false"
                     />
                 </q-tabs>
```

Comment:
Please, evaluate all the `console.error` messages below this point

Reply:
Wow!! there were so many XD

fixed